corehq/apps/domain/decorators.py

```diff
@@ -225,6 +234,28 @@ def _inner(request, *args, **kwargs):
     return _inner
 
 
+def check_lockout(fn):
+    @wraps(fn)
+    def _inner(request, *args, **kwargs):
+        username = _get_username_from_request(request)
+        user = CouchUser.get_by_username(username)
+        if user and user.is_web_user() and user.is_locked_out():
```

Comment:
This lockout condition block is being used in multiple places, if it is replaced by a simple function or method making changes to this condition latter can be done in just one place.